spec/services/provider_interface/diversity_data_by_provider_spec.rb

```diff
@@ -15,7 +15,7 @@ module ProviderInterface
 
     describe '#ethnicity_data' do
       it 'returns the ethnicity data for the provider' do
-        create(:application_form, recruitment_cycle_year: RecruitmentCycle.current_year, equality_and_diversity: { 'ethnic_group' => 'Mixed or multiple ethnic groups' }, application_choices: [create(:application_choice, :offered, provider_ids: [provider.id])])
+        create(:application_form, submitted_at: Time.zone.now, recruitment_cycle_year: RecruitmentCycle.current_year, equality_and_diversity: { 'ethnic_group' => 'Mixed or multiple ethnic groups' }, application_choices: [create(:application_choice, :offered, provider_ids: [provider.id])])
```

Comment:
Maybe use the `:minimum_info` trait here (and below)? Adds a bit more info than you need but makes sense if you are 'submitting' the form.